src/base.ts

```diff
@@ -13,6 +13,10 @@ import { NetworkName, NetworkNames } from "./networks";
 
 type TxError = { error: { reason: string } };
 
+function printCurrentNetwork(network: NetworkName) {
```

Comment:
this might cause unintended consequence in the dashboard. we parse stdout to get the result and we need to account for this new log [here](https://github.com/earthfast/earthfast-dashboard/blob/303a4394e4a7e77317002efe72b5c35ec8c62d05/server/src/cli.ts#L37) before we can release this

Reply:
hm ok, just removed by default for now